### PR TITLE
android-udev-rules: create adbusers group

### PR DESCRIPTION
**Pull request by rien**
If desired, I could also add a `post_install` function to run `udevadm control --reload`?

### CODE REVIEW

File: srcpkgs/android-udev-rules/template [hunk 1]
```diff
@@ -1,14 +1,17 @@
 # Template file for 'android-udev-rules'
 pkgname=android-udev-rules
 version=20200613
-revision=1
+revision=2
 short_desc="Android udev rules list aimed to be the most comprehensive on the net"
 maintainer="Rien Maertens <rien.maertens@posteo.be>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/M0Rf30/android-udev-rules"
+depends="xbps-triggers"
```
Comment (ericonr) [inline srcpkgs/android-udev-rules/template:9]: `xbps-triggers` is a dependency of `xbps` already, no need to have it here.

**Comment by ericonr** [comment]
Running `udevadm` is reasonable only if we add a trigger for it, IMO.

**Comment by rien** [comment]
Sure, I can create such a trigger based on the `hwdb.d-dir` trigger. Should I create a separate PR?

**Comment by ericonr** [comment]
I think so. Discussion can happen there, otherwise this PR can be held in limbo for a while.

**Comment by ericonr** [comment]
I'm not 100% on it needing to exist, btw. It's mostly for discussion.

**Comment by rien** [comment]
There are 33 lines matching `grep -r 'vinstall.*\.rules.*udev/rules.d' srcpkgs/`, which could possibly use this kind of trigger.

**Comment by ericonr** [comment]
If you look into xlocate, it's actually 101 packages with files in `/usr/lib/udev/rules.d`. Still, I'm not against it so I'd say you should go for it!

**Comment by Duncaen** [comment]
I don't think the INSTALL.msg is a good idea, its shown on each update, is not a good place to document things and users already ignore them and adding more would make users less likely to read them if there is actually a important message.

**Comment by sgn** [comment]
I'll push this out first, the udev rules triggers will be discussed more.

**Pull request closed without merging**